README.md

```diff
@@ -13,10 +13,15 @@ Copy `.env.example` to `.env` and put in your [Etherscan API Key](https://ethers
 To download all the test files and contracts for the challenges specified on `challenges.ts`, run:
 
 ```
-yarn install-challenges
+node install.js
```

Comment:
umm, why? there's no yarn? as an option we can just use npm (same question for all the `node install.js`)

Reply:
Because I didn't understand why we wanted to wrap a node command like that. 

node install / node install.js => yarn install-challenges? not sure what is the benefit of that